draft-liu-teas-transport-network-slice-yang.txt

```diff
@@ -0,0 +1,1568 @@
+   "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and
+   "OPTIONAL" in this document are to be interpreted as described in BCP
+   14 [RFC2119] [RFC8174] when, and only when, they appear in all
+   capitals, as shown here.
+
+   The terminology for describing YANG data models is found in
+   [RFC7950].
+
+
+
+
+
+
+Liu, et al.              Expires 13 August 2023                 [Page 3]
+
+Internet-Draft      Network Slice Topology Data Model      February 2023
+
+
+1.2.  Tree Diagram
+
+   Tree diagrams used in this document follow the notation defined in
+   [RFC8340].
+
+1.3.  Prefixes in Data Node Names
+
+   In this document, names of data nodes and other data model objects
+   are prefixed using the standard prefix associated with the
+   corresponding YANG imported modules, as shown in Table 1.
+
+             +==========+=======================+===========+
+             | Prefix   | YANG Module           | Reference |
+             +==========+=======================+===========+
+             | yang     | ietf-yang-types       | [RFC6991] |
+             +----------+-----------------------+-----------+
+             | inet     | ietf-inet-types       | [RFC6991] |
+             +----------+-----------------------+-----------+
+             | nt       | ietf-network-topology | [RFC8345] |
+             +----------+-----------------------+-----------+
+             | nw       | ietf-network-topology | [RFC8345] |
+             +----------+-----------------------+-----------+
+             | tet      | ietf-te-topology      | [RFC8795] |
+             +----------+-----------------------+-----------+
+             | te-types | ietf-te-types         | [RFC8776] |
+             +----------+-----------------------+-----------+
+             | ns-topo  | ietf-ns-topo          | RFCXXXX   |
+             +----------+-----------------------+-----------+
+
+             Table 1: Prefixes and Corresponding YANG Modules
+
+   RFC Editor Note: Please replace XXXX with the RFC number assigned to
+   this document.  Please remove this note.
+
+2.  Modeling Considerations
+
+   An IETF network slice is modeled as network topology defined in
+   [RFC8345], with augmentations.  A new network type "network-slice" is
+   defined in this document.  When a network topology data instance
+   contains the network-slice network type, it represents an instance of
+   an IETF network slice topology.
+
+2.1.  Relationships to Related Topology Models
+
+   There are several related YANG data models that have been defined in
+   IETF.  Some of these are:
+
+   Network Topology Model: Defined in [RFC8345].
+
+
+
+Liu, et al.              Expires 13 August 2023                 [Page 4]
+
+Internet-Draft      Network Slice Topology Data Model      February 2023
+
+
+   Network Slicing Model: Defined in
+   [I-D.ietf-teas-ietf-network-slice-nbi-yang].
+
+   OTN Slicing: Defined in [I-D.ietf-ccamp-yang-otn-slicing].
+
```

Comment:
Should we also consider NRP model?

Reply:
The NRP model is separate from the NS topology model.  It may be good to add a sentence about their relationship.